services/auth-service/.env.example

```
@@ -0,0 +1,3 @@
+PORT=5002
+MONGODB_URI=mongodb+srv://rishi3112:rishi3112@cluster0.yifopvp.mongodb.net/?appName=Cluster0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGODB_URI key should go before the PORT key

(UnorderedKey)